Sources/reactant-ui/GenerateCommand.swift

```diff
@@ -147,7 +147,7 @@ class GenerateCommand: Command {
 
             let xml = SWXMLHash.parse(data)
 
-            let node = xml["Component"].element!
+            guard let node = xml["Component"].element else { continue }
```

Comment:
Is it possible to show a warning here?

Reply:
Oh yeah the PR name was false advertising. Not yet as `Logger` is not available in Generator as of now, I will add a warning when it is.